src/components/common/RichText.tsx

```diff
@@ -55,12 +55,17 @@ type RichTextImageProps = {
 
 const StyledRichText = styled.div`
   .responsive-object {
-    padding-bottom: 0 !important;
+    position: relative;
     width: 100%;
-    max-width: ${(props) => props.theme.breakpointSm};
+    height: 0;
+    overflow: hidden;
   }
+
   .responsive-object iframe {
     aspect-ratio: 16 / 9;
+    position: absolute;
+    top: 0;
+    left: 0;
     width: 100%;
     height: 100%;
   }
```

Comment:
It's hard to tell without testing this, but won't this cause the `responsive-object` container to have a zero height so that following elements would overlap it?